h2o-automl/src/main/java/ai/h2o/automl/AutoMLBuildSpec.java

```diff
@@ -55,13 +59,95 @@ public AutoMLBuildControl() {
     public String export_checkpoints_dir = null;
   }
 
+  public static final class AutoMLStoppingCriteria extends Iced {
+
+    public static final int AUTO_STOPPING_TOLERANCE = -1;
+
+    private RandomDiscreteValueSearchCriteria _searchCriteria;
+    private double max_runtime_secs_per_model = 0;
+
+    public AutoMLStoppingCriteria() {
+      super();
+      _searchCriteria = new RandomDiscreteValueSearchCriteria();
+    }
+
+    public double max_runtime_secs_per_model() {
+      return max_runtime_secs_per_model;
+    }
+
+    public void set_max_runtime_secs_per_model(double max_runtime_secs_per_model) {
```

Comment:
@seb-h2o minor, underscore instead of `this`.

Reply:
right, I added the underscores to this class in a previous PR, forgot the new one during the merge